src/generators/web/client/components/SearchBox/index.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from 'react';
+import { create, load } from '@orama/orama';
+import { OramaSearchButton, OramaSearchBox } from '@orama/react-components';
+import { useTheme } from '../../hooks/useTheme.mjs';
+import { themeConfig } from './config.mjs';
+
+const MDXSearchBox = () => {
+  const [colorScheme] = useTheme();
+  const [client, setClient] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const initializeOrama = async () => {
+      try {
+        const db = create({
+          schema: {},
+        });
+
+        // Set the client immediately so it's available
+        setClient(db);
+
+        // Then fetch and load the data
+        const response = await fetch('/orama-db.json');
+        if (response.ok) {
+          load(db, await response.json());
+        }
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    initializeOrama();
+  }, []);
+
+  return (
+    <>
+      <OramaSearchButton
+        style={{ flexGrow: 1 }}
+        colorScheme={colorScheme}
+        themeConfig={themeConfig}
+        aria-label="Search documentation"
+      >
+        Search documentation
```

Comment:
```suggestion
        Search in documentation
```

maybe little bit better